app/main.py

```diff
@@ -1,6 +1,23 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()  # Завантажуємо змінні з .env
+
+API_KEY = os.getenv("API_KEY")
```

Comment:
Ensure that the `.env` file contains the `API_KEY` variable. If `API_KEY` is not set, the request will fail, and you might want to add error handling to check if `API_KEY` is `None`.